content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -1,73 +1,115 @@
-===========================
-Process returns and refunds
-===========================
+===================
+Returns and refunds
+===================
 
-The :guilabel:`Sales` app provides two different ways to process returns based on whether an
-invoice has been sent or not.
+The Odoo *Sales* app provides two different ways to process returns. The method used depends on
+whether or not an invoice has been sent.
 
 Before invoicing
 ================
 
-Returns are completed using *Reverse Transfers* when a customer decides to return a product before
-an invoice has been sent or validated.
+Returns are completed using *Reverse Transfers* when a customer decides to return a product
+**before** an invoice has been sent or validated.
 
 .. note::
-   In order to use Reverse Transfers, the :guilabel:`Inventory` app must also be installed.
+   In order to use Reverse Transfers, the :guilabel:`Inventory` app **must** be installed.
```

Comment:
Italicize app names when referring to them and the text is not present on the UI
```suggestion
   In order to use *Reverse Transfers*, the *Inventory* app **must** be installed.
```